plutus-benchmark/bls12-381-costs/test/9.6/bls12-381-costs.golden

```diff
@@ -2,144 +2,144 @@ Hash n bytestrings onto G1 and add points
 
     n     Script size             CPU usage               Memory usage
   ----------------------------------------------------------------------
-    0     115   (0.7%)        17294189   (0.2%)            3918   (0.0%) 
-   10     224   (1.4%)       693350079   (6.9%)           37278   (0.3%) 
-   20     334   (2.0%)      1369405969  (13.7%)           70638   (0.5%) 
-   30     444   (2.7%)      2045461859  (20.5%)          103998   (0.7%) 
-   40     554   (3.4%)      2721517749  (27.2%)          137358   (1.0%) 
-   50     664   (4.1%)      3397573639  (34.0%)          170718   (1.2%) 
-   60     774   (4.7%)      4073629529  (40.7%)          204078   (1.5%) 
-   70     884   (5.4%)      4749685419  (47.5%)          237438   (1.7%) 
-   80     994   (6.1%)      5425741309  (54.3%)          270798   (1.9%) 
-   90    1104   (6.7%)      6101797199  (61.0%)          304158   (2.2%) 
-  100    1214   (7.4%)      6777853089  (67.8%)          337518   (2.4%) 
-  110    1324   (8.1%)      7453908979  (74.5%)          370878   (2.6%) 
-  120    1434   (8.8%)      8129964869  (81.3%)          404238   (2.9%) 
-  130    1544   (9.4%)      8806020759  (88.1%)          437598   (3.1%) 
-  140    1654  (10.1%)      9482076649  (94.8%)          470958   (3.4%) 
-  150    1764  (10.8%)     10158132539 (101.6%)          504318   (3.6%) 
+    0     115   (0.7%)        53556222   (0.5%)            3918   (0.0%) 
```

Comment:
What happened to this?

Reply:
> What happened to this?

Do you mean why did the number for 0 get bigger?  If so, it's probably because that's a case where it's processing an empty list.  Some of the builtins have got  cheaper, but others have got more expensive, at least for certain inputs.  My guess is that there's some extra constant cost that gets swallowed up when it actually uses the BLS operations, which _have_ got cheaper.